clvm-derive/src/helpers.rs

```diff
@@ -20,20 +22,21 @@ impl fmt::Display for Repr {
     }
 }
 
-pub struct ClvmDeriveArgs {
-    pub repr: Repr,
+#[derive(Default)]
+pub struct ClvmAttr {
+    pub repr: Option<Repr>,
+    pub untagged: bool,
```

Comment:
Meaning that the discriminator of the enum (the numeric value) is not used as a prefix when encoding. In other words, the enum is used as a wrapper that checks each of the variants in order instead of the first value first.

Maybe there's a better name?